### PR TITLE
Add ActionContainer, LayoutCustomizationCard and PrepaidCard components 

### PR DESCRIPTION
**Pull request by burieberry**
## Boxel::ActionContainer component CS-103

action-container-component

#### Form view:
layout-data-entry

layout-selection

#### Memorialized / Display view:
layout-memorialized

### 2- PrepaidCard (card template)
prepaid-card

### CODE REVIEW

File: app/components/boxel/action-container/usage.hbs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  <:example>
+    <Boxel::ActionContainer
+      @header={{this.header}}
+      @prompt={{this.prompt}}
+      @incompleteActionLabel={{this.incompleteActionLabel}}
+      @completeActionLabel={{this.completeActionLabel}}
+      @isComplete={{this.isComplete}}
+      @onClickButton={{fn (mut this.isComplete) (not this.isComplete)}}
+    >
+
```
Comment (lukemelia) [inline app/components/boxel/action-container/usage.hbs]: Maybe add a little placeholder HTML here to visually indicate that the block is yielded.

File: app/components/boxel/action-container/usage.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/* eslint-disable ember/no-empty-glimmer-component-classes */
```
Comment (lukemelia) [inline app/components/boxel/action-container/usage.js]: This comment should not be needed.

File: app/components/card-pay/prepaid-card-issuance/layout-customization-card/display-view/index.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
+.layout-display-view {
+  --preview-card-width: 230px;
+  --preview-card-height: 145px;
+  --headerColor: var(--boxel-lime);
+  --headerPattern: var(--boxel-light);
```
Comment (lukemelia) [inline app/components/card-pay/prepaid-card-issuance/layout-customization-card/display-view/index.css]: Mixed casing of css vars. Do we have a standard convention?

**Review by lukemelia — APPROVED** [review]
A few minor comments, otherwise, ready to go! :shipit:

File: app/components/card-pay/prepaid-card-issuance/layout-customization-card/display-view/index.css (hunk 3 above)
Comment (burieberry) [reply app/components/card-pay/prepaid-card-issuance/layout-customization-card/display-view/index.css]: Dasherized is good. Fixed

**Pull request closed without merging**